modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/KaleoDesignerEditWorkflow.path

```diff
@@ -320,7 +320,7 @@
 </tr>
 <tr>
 	<td>VIEW_FORM_NODE</td>
-	<td>//div[@class='diagram-node-label' and .='${key_nodeNodeLabel}']/../div/div/div</td>
+	<td>//div[contains(@class,'base-node')]//div[contains(@class,'node')]//div[contains(.,'${key_nodeNodeLabel}')]</td>
```

Comment:
The path identifies the node by its Node Label as the test requires.
But it may be better to use the Name Node since it is the node identifier (unique attribute for each node). For this, it would be necessary to refactor some tests.